cluster/src/main/scala/org/apache/spark/sql/streaming/SnappyStreamingContextFactory.scala

```diff
@@ -61,13 +56,18 @@ class SnappyStreamingContextFactory extends SparkContextFactory {
     new SnappyStreamingContext(LeadImpl.getInitializingSparkContext,
       Milliseconds(interval)) with ContextLike {
 
+      private val addedJars = scala.collection.mutable.ArrayBuffer.empty[String]
```

Comment:
@rishitesh I think there is some confusion here. I am not asking if two StreamingContexts can be started which obviously cannot be due to Spark limitation. The question is: do we restrict users from submitting tasks to the same StreamingContext (e.g. gotten with getOrCreate) from different jobs? Are we documenting that do not attempt to submit tasks against the same context from two different jobs using REST/JDBC API?